Improve documentation (#122)
radicle-dev/radicle-upstream: proxy/README.md

# Proxy

A background service that implements all business logic tying together the

> radicle protocol and radicle registry. It provides a GraphQL API to the

Reviewer: ```suggestion
Code Collaboration and Registry. It provides a GraphQL API to the
```

> frontend.

Reviewer: ```suggestion
ui.
```

### Run

To start up the binary you can run: `cargo run -- --source=memory`.
After that the API is served on `http://127.0.0.1:8080/graphql`.

### Testing

Before running the test suite, download the test fixtures:

```sh

> git submodule update --recursive

Reviewer: This part seems to be shared between all tests in the repo, so might be better placed in the root readme.

Author: I think it would make the docs more confusing. If we keep the Rust and JS development sections in their respective READMEs and extract the submodule update step to the top-level, then someone reading this testing section might miss that part and wonder why the tests are not working.

# Proxy

A background service that implements all business logic tying together the

> radicle protocol and radicle-registry. It provides a GraphQL API to the

Reviewer 2: ```suggestion
radicle protocol and radicle registry. It provides a GraphQL API to the
```

> fronend.

Reviewer 2: ```suggestion
frontend.
```
